src/libraries/TransientStorage.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.26;
+
+/// @title  TransientStorage
+library TransientStorage {
+    function tstore(bytes32 slot, address value) internal {
+        assembly {
+            tstore(slot, value)
+        }
+    }
+
+    function tstore(bytes32 slot, uint256 value) internal {
+        assembly {
+            tstore(slot, value)
+        }
+    }
+
+    function tloadAddress(bytes32 slot) internal returns (address value) {
```

Comment:
```suggestion
    function asAddress(bytes32 slot) internal returns (address value) {
```

Reply:
Likewise I think this is good to name explicitly